File: redcap_v15.7.6/API/examples/ruby/delete_users.rb
#!/usr/bin/env ruby

require 'json'
require 'curl'
require './settings.rb'
include Settings

fields = {
  :token => Settings::API_TOKEN,
  :content => 'user',
  :action => 'delete',
  :format => 'json',
  'users[0]' => 'test_user_47',
}

ch = Curl::Easy.http_post(
  Settings::API_URL,
  fields.collect{|k, v| Curl::PostField.content(k.to_s, v)}
)
puts ch.body_str

File: html/redcap_v15.7.6/API/examples/csharp/DeleteUsers.cs
using RestSharp;
namespace RedcapCSharpApiExamples
{
    public static class DeleteUsers
    {
        public static async Task<RestResponse> DeleteUsersAsync(string[] users)
        {
            try
            {
                // Setup
                var apiToken = Config.ApiToken;
                var uri = new Uri(Config.ApiUrl);

                // Create Request
                var client = new RestClient();
                var request = new RestRequest(uri, Method.Post);
                request.AddHeader("Content-Type", "application/x-www-form-urlencoded");
                request.AddParameter("token", apiToken);
                request.AddParameter("content", "user");
                request.AddParameter("action", "delete");
                request.AddParameter("format", "json");

                for (var i = 0; i < users.Length; i++)
                {
                    request.AddParameter($"users[{i}]", users[i]);
                }

                // Execute Request
                var response = await client.ExecuteAsync(request);
                return response;
            }
            catch (Exception e)
            {
                Console.WriteLine(e.Message);
                throw;
            }
        }
    }
}
